.ci/eslint-plugin-zotero-translator/lib/rules/license.js

```diff
@@ -26,20 +26,31 @@ module.exports = {
 				if (!options.mustMatch) throw new Error('mustMatch not set');
 
 				let firstComment = null;
+				let copyright = null;
 				for (const comment of context.getSourceCode().getAllComments()) {
 					if (comment.loc.start.line <= header.body.loc.end.line) continue; // skip decorator comments
 
 					if (comment.value.includes(options.mustMatch)) {
-						if (firstComment) {
+						if (firstComment || comment.loc.start.line > header.followingStatement.start.line) {
 							context.report({
 								loc: comment.loc,
 								message: 'Preferred to have license block at the top'
 							});
 						}
 						return;
 					}
+
+					if (comment.value.match(/copyright/i)) copyright = comment;
```

Comment:
Braces

Reply:
Braces added, but it passed the existing eslint rules.